src/Tasks/FileIO/WriteLinesToFile.cs

```diff
@@ -82,27 +83,32 @@ public override bool Execute()
 
                 try
                 {
+
+                    var directoryPath = Path.IsPathRooted(File.ItemSpec) ? Path.GetDirectoryName(File.ItemSpec) : Path.GetDirectoryName(Path.GetFullPath(File.ItemSpec));
```

Comment:
- use FileUtilities.NormalizePath instead of Path.GetFullPath
- since you may end up calculating the full path here, might as well store it and reuse it in the places where File.ItemSpec gets sent to IO calls. That way the IO calls would avoid doing their own GetFullPath.

Reply:
Ok - I will do that :)